src/components/list-option/list-option.css

```diff
@@ -17,7 +17,7 @@
 }
 
 .option:hover {
-  background-color: var(--selected-background); /* stylelint-disable-line mds/colors */
+  background-color: var(--selected-background) !important; /* stylelint-disable-line mds/colors */
```

Comment:
Just curious, how come we need this?

Reply:
In bigmaven, there is some CSS pollution that is clobbering `li:hover` :( A couple of teams already complained about the lack of the background-color and I opted to just `!important` to deal with it asap.